docs/integrations/prometheus/_index.rst

```diff
@@ -16,11 +16,14 @@
 | Determined 0.17.6+ |
 +--------------------+
 
-Discover how to enable a Grafana dashboard to monitor Determined hardware and system metrics on a
-cloud cluster, such as AWS or Kubernetes. Determined provides a Prometheus endpoint that contains
-mappings between internal task, GPU, and container definitions, which are used by Prometheus to
-collect relevant metrics on a cluster running Determined. The endpoint is not enabled by default but
-can be enabled in the master configuration file.
+Discover how to enable a Grafana dashboard to monitor Determined hardware and system metrics.
+Determined provides a Prometheus endpoint that contains mappings between internal task, GPU, and
+container definitions, which are used by Prometheus to collect relevant metrics on a cluster running
+Determined. The endpoint is not enabled by default but can be enabled in the master configuration
```

Comment:
prometheus should be enabled by default now

Reply:
Nice catch! Thank you!

I inverted the sentence `The endpoint is enabled by default but can be disabled in the master configuration
file.`